modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/FailoverEndpointSerializer.java

```diff
@@ -50,6 +50,10 @@ protected OMElement serializeEndpoint(Endpoint endpoint) {
 
         serializeCommonAttributes(endpoint,endpointElement);
 
+        if (failoverEndpoint.isBuildMessageAtt()) {
+            failoverElement.addAttribute("buildMessage", Boolean.toString(failoverEndpoint.isBuildMessageAtt()), null);
```

Comment:
As "buildMessage" has been used multiple places, shall we define it as a constant?

Reply:
Fixed